apps/dashboard/app/(app)/desktop-sidebar.tsx

```diff
@@ -113,6 +114,12 @@ export const DesktopSidebar: React.FC<Props> = ({ workspace, className }) => {
       tag: <Tag label="internal" />,
       hidden: !workspace.features.successPage,
     },
+    {
+      icon: DatabaseZap,
+      href: "/semantic-cache",
+      label: "Semantic Cache",
+      active: segments.at(0) === "semantic-cache",
+    },
```

Comment:
let's hide this for now if we want to merge, see the link above

Reply:
Done